src/publications.md

```diff
@@ -6,7 +6,15 @@ Title: Publications
 The following publications relied on DataJoint open-source software for data analysis.  If your work uses DataJoint or DataJoint Elements, please cite the respective
 [manuscripts and RRIDs](./citation.md).
 
-## 2022 <!-- 21 -->
+## 2023 <!-- 3 -->
```

Comment:
Maybe we should update the external links on this page as well to be more in MkDocs format.

Reply:
Updated!